x-pack/plugins/searchprofiler/server/routes/profile.ts

```diff
@@ -56,6 +57,49 @@ export const register = ({ router, getLicenseStatus, log }: RouteDependencies) =
         log.error(err);
         const { statusCode, body: errorBody } = err;
 
+        return response.customError({
+          statusCode: statusCode || 500,
+          body: errorBody
+            ? {
+                message: errorBody.error?.reason,
+                attributes: errorBody,
+              }
+            : err,
+        });
+      }
+    }
+  );
+  router.get(
+    {
+      path: `${API_BASE_PATH}/has_indices`,
+      validate: false,
+    },
+    async (ctx, _request, response) => {
+      const currentLicenseStatus = getLicenseStatus();
+      if (!currentLicenseStatus.valid) {
+        return response.forbidden({
+          body: {
+            message: currentLicenseStatus.message!,
+          },
+        });
+      }
+
+      try {
+        const client = (await ctx.core).elasticsearch.client.asCurrentUser;
+        const resp = await client.cat.indices({ format: 'json' });
+
+        const hasIndices = resp.length > 0;
+
+        return response.ok({
+          body: {
+            ok: true,
```

Comment:
I dont think we need the `ok: true` anymore

Reply:
i missed that part of your previous comment 😓